src/scripts/tbtcv2-rewards/rewards.ts

```diff
@@ -525,21 +507,16 @@ async function getAuthorization(
 function authorizationForRewardsInterval(
```

Comment:
Should we remove the comment above? lines 493 to 506:

https://github.com/threshold-network/merkle-distribution/blob/055f8499962eeb798f9532e4095f4590762358f6/src/scripts/tbtcv2-rewards/rewards.ts#L493-L506

Reply:
Right, this should be removed. Here [e2b0a63](https://github.com/threshold-network/merkle-distribution/pull/89/commits/e2b0a63be04cb80f4d11d37c5b9a04f23a14fc85)